packages/augur-ui/src/modules/common/buttons.styles.less

```diff
@@ -1006,6 +1006,70 @@
   }
 }
 
+.BettingBackLayButton {
+  .Button;
+
+  background: transparent;
+  color: var(--color-primary-text);
+  padding: @size-6 @size-12;
+
+  > div:first-of-type {
+    .text-12-bold;
+
+    color: var(--color-primary-text);
+    text-align: center;
+  }
+
+  > div:last-of-type {
+    .text-12;
+
+    color: var(--color-primary-text);
+    text-align: center;
+  }
+
+  &.Back {
+    border: @size-1 solid var(--color-outcome-five);
+
+    &:disabled {
+      background: var(--color-inactive-text);
+      opacity: 0.15;
+    }
+  
+    &:hover:enabled {
+      background: var(--color-outcome-five-40);
+    }
+  
+    &:focus:enabled {
+      background: var(--color-outcome-five-40);
+    }
+  
+    &:active:enabled {
+      background: var(--color-outcome-five-40);
+    }
+  }
+
+  &.Lay {
+    border: @size-1 solid var(--color-outcome-two);
+
+    &:disabled {
+      background: var(--color-inactive-text);
```

Comment:
`--color-negative-15` and no opacity change

Reply:
Updated. Thank you.